internal/docs/resources.go

```diff
@@ -0,0 +1,84 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package docs
+
+import _ "embed"
+
+// Input definitions
+
+//go:embed _static/inputs/aws-cloudwatch.yml
```

Comment:
Is it also possible to embed a whole directory. In case this helps, maybe when adding new inputs in the future.
```
//go:embed _static/inputs
var inputResources embed.FS
```